pyanaconda/storage_utils.py

```diff
@@ -347,6 +348,20 @@ def verify_swap(storage, constraints, report_error, report_warning):
                                  "it will significantly improve performance "
                                  "for most installations."))
 
+    else:
+        swap_space = sum((device.size for device in swaps), Size(0))
+        disk_space = sum((device.size for device in storage.mountpoints.values()), Size(0))
+        recommended = swap_suggestion(disk_space=disk_space, quiet=True)
```

Comment:
Shouldn't we show warning or raise exception when `disk_space` will be `Size(0)` or is it expected behavior?